wagtail/admin/tests/test_contentstate.py

```diff
@@ -436,6 +436,21 @@ def test_block_element_in_paragraph(self):
             }
         })
 
+    def test_br_element_in_paragraph(self):
+        converter = ContentstateConverter(features=[])
+        result = json.loads(converter.from_database_format(
+            '''
+            <p>before<br/>after</p>
+            '''
+        ))
+        self.assertContentStateEqual(result, {
+            'entityMap': {},
+            'blocks': [
+                {'key': '00000', 'inlineStyleRanges': [], 'entityRanges': [], 'depth': 0, 'text': 'before\nafter',
+                 'type': 'unstyled'}
+            ],
+        })
+
```

Comment:
I believe we also need to test what happens if the `br` isn't inside a `p` (or any other block) tag. That will never happen with content coming from Draftail but I've seen [some weird things with Hallo.js and line breaks](https://github.com/thibaudcolas/draftjs_exporter_wagtaildbhtml#line-breaks), so this needs further consideration.

Reply:
I know this was old but this seemed like an odd test, since normally `<br>` is not written as `<br/>`